lib/compat/wordpress-6.2/default-filters.php

```diff
@@ -17,11 +17,6 @@
  * @package gutenberg
  */
 
-/**
- * Note for backport: we should also remove the existing filters:
- *
- * > add_action( 'switch_theme', array( 'WP_Theme_JSON_Resolver', 'clean_cached_data' ) );
```

Comment:
We should not remove these filters when backporting to core, so I've removed the note.

Each public method has its own conditions for invalidating the cache (this one does not require invalidation when a plugin is updated, unlike the [settings](https://github.com/WordPress/gutenberg/pull/45372#issuecomment-1296859561) or [styles](https://github.com/WordPress/gutenberg/pull/45679#discussion_r1018520188)), so we should have atomic clean operations instead of a whole.